From fb5056fea9559b3ec3bb61967a112b6c793b700d Mon Sep 17 00:00:00 2001
From: Khem Raj <raj.khem@gmail.com>
Date: Tue, 8 Jan 2013 21:33:34 -0800
Subject: [PATCH] include unistd.h to get ssize_t definitions

Fixes errors like

| In file included from plugins/mas.c:36:0:
| ./src/obex.h:39:1: error: unknown type name 'ssize_t'
| ./src/obex.h:49:1: error: unknown type name 'ssize_t'
| ./src/obex.h:50:1: error: unknown type name 'ssize_t'
| In file included from plugins/mas.c:38:0:
| ./src/mimetype.h:36:2: error: expected specifier-qualifier-list before 'ssize_t'
| In file included from plugins/mas.c:39:0:
| plugins/filesystem.h:24:1: error: unknown type name 'ssize_t'
| plugins/mas.c:550:1: error: unknown type name 'ssize_t'
| plugins/mas.c:557:1: error: unknown type name 'ssize_t'
| plugins/mas.c: In function 'any_read':
| plugins/mas.c:560:2: error: unknown type name 'ssize_t'

Upstream-Status: Pending

Signed-off-by: Khem Raj <raj.khem@gmail.com>

---
 plugins/mas.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/plugins/mas.c b/plugins/mas.c
index 1b18059..b519507 100644
--- a/plugins/mas.c
+++ b/plugins/mas.c
@@ -30,6 +30,7 @@
 #include <glib.h>
 #include <fcntl.h>
 #include <inttypes.h>
+#include <unistd.h>
 
 #include <gobex/gobex.h>
 #include <gobex/gobex-apparam.h>
